lib/dash.coffee

```diff
@@ -55,7 +57,17 @@ plugin = module.exports =
       path = activeEditor.getPath()
       language = activeEditor.getGrammar().name
 
-    spawn('open', ['-g', @createLink(string, path, language)])
+    if platform == "win32"
+      # Exec is used because spawn escapes arguments that contain double-quotes
+      # and replaces them with backslashes. This interferes with the ability to
+      # properly create the child process in windows, since windows will barf
+      # on an ampersand that is not contained in double-quotes.
+      quotedLink = '"' + @createLink(string, path, language) + '"'
+      exec('cmd.exe /c start "" ' + quotedLink, null)
```

Comment:
The ```/c``` option for cmd.exe is the functional equivalent to the ```-g``` flag for ```open``` on the Mac.   

From the cmd.exe help reference: 
```/C      Carries out the command specified by string and then terminates```

Keep in mind, that this logic only applies to win32 platform. I've left the logic for darwin and other platforms untouched with the if/then block. So on the Mac it should still execute ```spawn('open', ['-g', @createLink(string, path, language)])```